src/test/java/com/wikia/webdriver/elements/mercury/components/TopBar.java

```diff
@@ -115,4 +124,13 @@ public boolean isCloseIconVisible() {
       return false;
     }
   }
+
+  public boolean isFandomBarVisible() {
+    try {
+      return fandomBar.isDisplayed();
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)